addons/hr_holidays/models/hr_leave.py

```diff
@@ -377,6 +379,11 @@ def _onchange_leave_dates(self):
         else:
             self.number_of_days = 0
 
+    @api.depends('employee_id')
```

Comment:
Hello @LucasLefevre 

We should display alert only if employee is set (and mode is By Employee)

- We should only display alert if Employee is set.
![Screenshot from 2019-06-12 00-25-33](https://user-images.githubusercontent.com/16624719/59298661-a1e45180-8ca8-11e9-8e0b-878c8f4e0679.png)